test/models.jl/uni.jl

```diff
@@ -0,0 +1,38 @@
+using Test, Turing
+turnprogress(false)
+
+n_samples = 20_000
+mean_atol = 0.25
+var_atol = 1.0
+
+for dist ∈ [Normal(0, 1), Gamma(2, 3), InverseGamma(2, 3), Beta(2, 1), Cauchy(0, 1)]
+
+    @testset "$(string(typeof(dist)))" begin
```

Comment:
Might be an idea to provide more than just type information for printing here. Probable that we'll eventually want more than one test per distribution to ensure that they work for a range of parameters, and only printing type info. will make it hard to see what tests are doing what.

Reply:
Good point. I'll make the change.

Reply:
I ended up with adding an info inside the `@testset` body as the full information is too long in the test table.